go.mod

```diff
@@ -259,3 +259,5 @@ require (
 )
 
 go 1.18
+
+replace github.com/ipfs/go-ipns => github.com/ipfs/go-ipns v0.2.1-0.20220912193459-b655f6b85fee
```

Comment:
- [ ] Remove and switch to release with  https://github.com/ipfs/go-ipns/pull/41